docs/configuration.md

```diff
@@ -118,6 +118,28 @@ This is the path to the app's instrumentation apk. Supports wildcard characters.
 ### additionalTestApks
 Paths to additional test configurations.
```

Comment:
maybe it's a good chance to expand a bit on the relationship between `app` and `test` entries, and most importantly mention the "ordering aspect" of this list.